components/Cards/src/CardThemer/MDCCardThemer.m

```diff
@@ -0,0 +1,59 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCCardThemer.h"
+
+#import "MaterialCards+ColorThemer.h"
+
+static const MDCShadowElevation kElevation = 0.f;
+static const MDCShadowElevation kSelectedElevation = 4.f;
+static const MDCShadowElevation kHighlightedElevation = 8.f;
+static const CGFloat kBorderWidth = 1.f;
+
+@implementation MDCCardThemer
+
++ (void)applyScheme:(nonnull id<MDCCardScheming>)scheme
+             toCard:(nonnull MDCCard *)card {
+  [MDCCardsColorThemer applySemanticColorScheme:scheme.colorScheme toCard:card];
+}
+
++ (void)applyScheme:(nonnull id<MDCCardScheming>)scheme
+         toCardCell:(nonnull MDCCardCollectionCell *)cardCell {
+  [MDCCardsColorThemer applySemanticColorScheme:scheme.colorScheme toCardCell:cardCell];
+}
+
++ (void)applyOutlineVariantWithScheme:(nonnull id<MDCCardScheming>)scheme
+                               toCard:(nonnull MDCCard *)card {
+  [card setShadowElevation:kElevation forState:UIControlStateNormal];
```

Comment:
I think we need to reset states here as well.

Reply:
Hmm, so I hesitated to reset here since these values are digits and cannot be reset to nil, only set to 0. I suppose I could set all states to kElevation except for the highlighted state